### PR TITLE
adapted to the current sw360portal which now uses maven-war-plugin

### PR DESCRIPTION
**Pull request by alex-evo**
see https://github.com/sw360/sw360portal/pull/241

### CODE REVIEW

File: sw360-single/sw360-install.sh [hunk 1]
```diff
@@ -177,27 +175,25 @@ if $doClean; then
...
-if $doFrontend; then
-    echo "Start of frontend deployment of sw360"
-    cd $wd/frontend
-    mvn install -P deploy $mavenParameters
-elif $doPortlets; then
-    echo "Start of frontend deployment of sw360"
-    cd $wd/frontend/sw360-portlet
+if $doAll; then
     mvn install -P deploy $mavenParameters
+else
```
Comment (maximilianhuber) [inline sw360-single/sw360-install.sh]: I am not sure whether in this case on should also do the `mvn install` to compile libs. Only doploying backend without previous compiling of lib-datahandler and co won't work.

Comment (alex-evo) [inline sw360-single/sw360-install.sh]: It seems to me like it was the same functionality before my change. That is, if you set -B option (or -P, or -F, for that matter), lib-datahandler won't be compiled.

Comment (maximilianhuber) [inline sw360-single/sw360-install.sh]: I though, one could choose to do the libs and the frontend but no backend.

I thing nobody will miss this feature.

**Comment by maximilianhuber** [comment]
The code was tested, it works fine. This can be merged.

**Pull request merged**